### PR TITLE
docs(devtools): changed port to 8001

### PR DESCRIPTION
**Pull request by NaokiHaba**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?

Adjusted description to match ListenPort of 8001 in screenshot

Due to the lack of a matching screenshot, we couldn't adjust for variations in the port number shown. However, we believe this should have minimal impact on the reader's understanding.

Issue Number: N/A

## What is the new behavior?

Changed the port number to 8001 and updated all references within the page. Note that some screenshots still show port 8000, but we believe this should have little impact on readers. Please review

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: content/devtools/overview.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ async function bootstrap() {
   const app = await NestFactory.create(AppModule, {
     snapshot: true,
   });
-  await app.listen(3000);
+  await app.listen(8001);
```
Comment (NaokiHaba) [inline content/devtools/overview.md:12]: Updated the relevant image to show port 8001


https://null/null/blob/2fa2a98c1c8fa7a3387d5b319b4754bf45bbd5a6/content/devtools/overview.md#L89

Comment (kamilmysliwiec) [reply content/devtools/overview.md:12]: The application can run on any port you wish (3000 is fine). `DevtoolsHttpModule` exposes another HTTP server which by default uses 8000 (so there're basically 2 HTTP servers)

File: content/devtools/overview.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ Once the `DevtoolsModule` is imported and your application is up and running (`n
 
 <figure><img src="/assets/devtools/modules-graph.png" /></figure>
 
-So as we can see, `DevtoolsModule` makes your application expose an additional HTTP server (on port 8000) that the Devtools application will use to introspect your app.
+So as we can see, `DevtoolsModule` makes your application expose an additional HTTP server (on port 8001) that the Devtools application will use to introspect your app.
```
Comment (kamilmysliwiec) [inline content/devtools/overview.md:47]: 8000 is the default

File: content/devtools/overview.md [hunk 3]
```diff
@@ -129,7 +129,7 @@ To see the auto-generated audit - errors/warnings/hints that the application cam
 To save a serialized graph to a file, use the following code:
 
 ```typescript
-await app.listen(3000); // OR await app.init()
+await app.listen(8001); // OR await app.init()
```
Comment (kamilmysliwiec) [inline content/devtools/overview.md:132]: https://github.com/nestjs/docs.nestjs.com/pull/2672/files#r1142973988

**Pull request closed without merging**